lib/dialects/postgres/query-generator.js

```diff
@@ -575,7 +576,7 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
 
     for (const key in attributes) {
       const attribute = attributes[key];
-      result[attribute.field || key] = this.attributeToSQL(attribute, options);
+      result[attribute.field || key] = this.attributeToSQL(attribute, Object.assign({ key }, options || {}));
```

Comment:
Why this change?

Reply:
To create a comment SQL statement in pg, it was necessary to have table name and column the comment is referring to. That line just adds the column name to options.